src/components/TransformtoEcommerce.js

```diff
@@ -0,0 +1,128 @@
+import { Button } from "@newfold/ui-component-library";
+import { __ } from "@wordpress/i18n";
+import { ReactComponent as CheckMarkIcon } from "../icons/check.svg";
+import { ReactComponent as LockIcon } from "../icons/lockicon.svg";
+import { ReactComponent as TransformStore } from "../icons/transform-store.svg";
+import { ReactComponent as WonderCartIcon } from "../icons/wondercart.svg";
+import { ReactComponent as YithPluginsIcon } from "../icons/yithplugins.svg";
+import { NewfoldRuntime } from "../sdk/NewfoldRuntime";
+
+
+
+export function TransformtoEcommerce() {    
+    return (
+        <div className={"nfd-app-section-container nfd-bg-white nfd-w-full nfd-rounded-lg " + (!(NewfoldRuntime.isWoo) ? 'nfd--mt-12' : '')} id="nonecommerce-features-wrapper">        
+            <div className="nfd-app-section-content nfd-border nfd-rounded-lg nfd-px-8 nfd-pt-0 nfd-pb-8 nfd--mt-2 nfd-items-start" style={{ borderColor: "#E2E8F0" }}>
+                <div className="nfd-flex nfd-flex-wrap nfd-gap-x-10">
+                    <div className="nfd-flex-none nfd-mt-8 nfd-w-40">
+                        <TransformStore />
+                    </div>
+                    <div class="nfd-flex-1 nfd-mt-5 nfd-pt-0.5">
+                        <h1 className="nfd-text-2xl nfd-leading-9 nfd-font-medium" style={{ color: "#111729" }}>
```

Comment:
can we use class instead of inline style  ?

Reply:
done